swift/Tests/HDWalletTests.swift

```diff
@@ -53,6 +53,15 @@ class HDWalletTests: XCTestCase {
         XCTAssertEqual("bc1qumwjg8danv2vm29lp5swdux4r60ezptzz7ce85", address)
     }
 
+    func testDeriveDigiByte() {
+        let bitcoin = CoinType.digiByte
+        let wallet = HDWallet.test
+        let key = wallet.getKeyForCoin(coin: bitcoin)
```

Comment:
`(coin: .digiByte)`

Reply:
Done in 58f703b0ae909c814e1803b3e7e99ccc83e61fbc